src/XrmContext/Interpretation/InterpretEntityMetadata.fs

```diff
@@ -61,7 +61,7 @@ let interpretAttribute deprecatedPrefix entityNames (e:EntityMetadata) (a:Attrib
   let aType = a.AttributeType.GetValueOrDefault()
 
   if not canGet ||
-    aType = AttributeTypeCode.Virtual ||
+    aType = AttributeTypeCode.Virtual && not (a :? MultiSelectPicklistAttributeMetadata) ||
```

Comment:
I would suggest to instead split the handling of virtual attributes into a separate if-branch. 
Having to perform the same type-checks of the virtual attribute in different places of the code, makes it harder to ensure correctness (i.e. if more virtual attributes need to be handled in the future).

Reply:
agreed, RN I don't have the time for it.
 it shloud go on some todo list (and hopefully taken care of one day).